homework.2/homework.2.playground/Contents.swift

```diff
@@ -0,0 +1,39 @@
+// Homework.2
+
+import Darwin
+
+let valueOne = 2.5
+let valueTwo = 9.7
+let valueThree = 6.9
+let valueFour = 8.2
+print("Original values - \(valueOne), \(valueTwo), \(valueThree), \(valueFour)\n")
+
+// Separating the integer part from values
+
+let valueOneWholePart = Int(valueOne)
+let valueTwoWholePart = Int(valueTwo)
+let valueThreeWholePart = Int(valueThree)
+let valueFourWholePart = round(valueFour)
```

Comment:
It's better to cast `valueFourWholePart` to `Int` here than multiple times later: `Int(round(valueFour))`

Reply:
I agree, in this case, this technique does not make sense. Used it as another solution :)

Reply:
Yeah, that's great that you explored different solutions. It's also another good skill😊 Because there's no only right solution for each problem so it's nice to see that. You've just missed that there's a way that we can make this solution a little simpler and that's nothing wrong with that